### DISCUSSION TITLE
Regarding the problem of not being able to activate the professional version after payment and not being able to receive the email verification code

### DISCUSSION THREAD

**Discussion by xiaoxiong713** [getlantern/lantern-forum-cn#674] [Q&A] [answered]
After I successfully paid through Alipay, I cannot update to the professional version because I cannot receive the email verification code. After updating to Mac 8.0.0 version, I cannot even activate the professional version through email. When I click Submit Email, the software directly reports an error. I hope to solve the problem or get a refund, thank you! @Mike777Khoo 

I submitted the payment screenshot and email address through the link

**Comment by MaxMKMax** [comment, marked as answer]
@xiaoxiong713 
Hi xiaoxiong, sorry for the inconvenience caused. Don't worry our customer support already received your issues. They will help to proceed or solve the issues in the next 2-5 days. We will help you to solve the issues, hope you might not request for refund. Thanks!

We will handle your case properly and if you do not see any updates in the next 3 days, please follow with up and i will escalate the case for you to the person in charge for an immediate solution. Sorry for the inconvenience caused. Appreciate your patient in waiting. Thank you so much for supporting Lantern.

**Reply by xiaoxiong713** [reply]
I have received the refund，Thank you for your reply

**Reply by MaxMKMax** [reply]
@xiaoxiong713 You are welcome. Thank you for supporting Lantern. Hope to see you again in the future. Welcoming back to purchase Lantern pro version anytime and welcome you to use our free version all the time. Thanks for your previous support and feel free to reach me again if you need any assistance/help. Lantern is welcoming you back to be our user anytime.
